Restore missing features: haproxy and file manager

kbumsik opened the pull request:
Hi,
Since I have used this project on a daily basics (mainly for Airplay though) I noticed a few things:

* There is no convenient way to upload files to the persistent storage for Mopidy. File manager was added in #25 for this, but it was excluded in the `docker-composer.yml` since the project was migrated to multi-containers configuration.

* Although the Wiki [states that](https://github.com/balena-io-projects/boombeastic/wiki/Using-BoomBeastic-Emoji-drawing) I can access to the services using path like http://boombeastic_ip/**emoji/** , I cannot use those paths as haproxy is not enabled in `docker-composer.yml` as well.

This PR addresses those problems:

* Restore Filemanager and make Mopidy handle files better (update file changes).
* Restore haproxy.

It could be two separated PRs but the changes are rather small, so I decided to propose them in a single PR.

BTW, please consider to modify the permissions of the Wiki page so that users like me can fix some errors.

curcuz commented:
Hey @kbumsik thanks a lot for taking care of this, I've been very busy and this slipped under my radar :)

curcuz commented on proxy/haproxy.cfg:
```diff
@@ -16,12 +16,6 @@ frontend public
         use_backend mopidympd if { path_beg /mpd/ }
         use_backend emoji if { path_beg /emoji/ }
         default_backend mopidyhttp
-        # Any URL beginning with socket.io will be flagged as 'is_websocket'
```
this ensures every boombeastic service can interact with each other, for example when you connect a client via airplay, the airplay services requests the emoji service to display an airplay icon on the led matrix. Removing it will break these features

kbumsik commented:
@curcuz You must have been very busy, looking at the latest news from Balena :) Congrats on the Series B funding!
It was my mistake to forgot mentioning why I removed the websocket proxy. I removed it as `expose: "1337"` in the `docker-compose.yml` will do enough job.
